projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -558,10 +746,30 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
         return false;
     }
 
+    /**
+     * Returns the current function by which the different data items are compared.
```

Comment:
@alexalex972 This needs more explanation. Why do we compare items? Why would the developer need to set this option? What use case it resolves?

Reply:
@mpavlinov I suggest something like this, might be too long for a description so let me know what you think: 

Gets/Sets the function used to track changes in the items collection. This function can be set in scenarios where you want to optimize or customize the tracking of changes for the items in the collection.
The trackBy function takes the index and the current item as arguments and needs to return the unique identifier for this item.
By default the object references are compared, however this can be optimized if you have unique identifier value that can be used for the comparison instead of the object ref or if you have some other property values in the item object that should be tracked for changes.
This option is similar to ngForTrackBy.